local-test.sh

```diff
@@ -9,7 +9,9 @@ cd "$(dirname "$0")"
 LATEST_LINE=weekly
 : "${LINE:=$LATEST_LINE}"
 
-SAMPLE_PLUGIN_OPTS=-Dtest=InjectedTest
+if [[ -z "${SAMPLE_PLUGIN_OPTS:-}" ]]; then
```

Comment:
Can be simplified to

```suggestion
if [[ -z ${SAMPLE_PLUGIN_OPTS-} ]]; then
```